test/Project.toml

```diff
@@ -72,7 +72,6 @@ ClimaCorePlots = "0.2"
 ClimaCoreSpectra = "0.1"
 ClimaCoreTempestRemap = "0.3"
 ClimaCoreVTK = "0.7"
-CloudMicrophysics = "0.15.2"
```

Comment:
Should we add `0.15` back in?

Reply:
Sure, but 0.15 uses ClimaParameters 0.8 so I don't think it would get used